Rakefile

```diff
@@ -0,0 +1,6 @@
+# Add your own tasks in files placed in lib/tasks ending in .rake,
+# for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
```

Comment:
__[rubocop]__
* Metrics/LineLength: Line is too long. [90/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)